src/ontology/uberon-edit.obo

```diff
@@ -106123,8 +106123,9 @@ property_value: external_definition "Simple squamous epithelium that lines the c
 
 [Term]
 id: UBERON:0005751
-name: visceral layer of glomerular capsule
+name: glomerular visceral epithelium
 def: "The inner, or visceral, layer of the renal glomerular capsule, overlying the capillaries, composed of podocytes, and separated from the outer, or parietal, layer by the capsular (Bowman's) space[BTO]. Lies just above the thickened glomerular basement membrane and is made of podocytes. Beneath the visceral layer lie the glomerular capillaries[WP]. inner layer of the glomerular capsule surrounding the capillary endothelial cells and composed of highly specialized epithelial cells (podocytes) that are separated by filtration slits; the visceral layer is separated from the outer parietal layer by the urinary space (also known as the Bowman's or capsular space)[MP]." [BTO:0001515, MP:0011497]
```

Comment:
Please check capitalisation and phrasing starting at "inner layer of the glomerular capsule..."

Reply:
I agree this definition should be refined. Do you think the [WP] is a typo and should be [MP] instead? I don't see any reference for WP elsewhere in this class.

Reply:
@bvarner-ebi [WP] is definitely not a typo, when those [xx] appear in definitions they were usually derived automatically from other ontologies or resources. I strongly suspect that WP stands for Wikipedia. FYI, a list of abbreviations used in Uberon is in the 2012 paper here: https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-1-r5#abbreviations. WP definitely does not refer to the Worm Phenotype Ontology ;-)

Reply:
Thank you, @paolaroncaglia! :)